packages/roosterjs-editor-plugins/lib/plugins/CustomReplace/CustomReplace.ts

```diff
@@ -131,9 +146,8 @@ export default class CustomReplacePlugin implements EditorPlugin {
             const [sourceMatch, replacementMatch] = replacement.matchSourceCaseSensitive
                 ? [stringToSearch, replacement.sourceString]
                 : [lowerCaseStringToSearch, replacement.sourceString.toLocaleLowerCase()];
-
             if (
-                sourceMatch.substring(sourceMatch.length - replacementMatch.length) ==
+                sourceMatch.substring(sourceMatch.length - replacementMatch.length) ===
```

Comment:
Just curious, is there any case broken without this change?

Reply:
The tests cases didn't break, I think is because the replacementMatch and the sourceString are defined as strings.

Reply:
But its better to let the way it was, since I didn't test all scenarios